server/models/User.js

```diff
@@ -42,6 +45,19 @@ const userSchema = new mongoose.Schema({
 	]
 });
 
+// Strips sensitive information when returning user
+userSchema.methods.toJSON = function () {
+	const user = this;
+	const userObject = user.toObject();
```

Comment:
since `user` variable is only used once, you don't need to store it in `this` variable